.isort.cfg

```diff
@@ -1,4 +1,4 @@
 [settings]
 profile=black
 extend_skip_glob=.github/*,third-party-stubs/*,scripts/*
-known_first_party=dbt,dbt_adapters,dbt_common,dbt_extractor,dbt_semantic_interface
+known_first_party=dbt,dbt_adapters,dbt_common,dbt_extractor,dbt_semantic_interfaces
```

Comment:
Should `snowplow_tracker` also be included here?  We maintain our own fork of it (minimal-snowplow-tracker)

Reply:
That's a good question 🤔 I do not know. We can, but it feels in a gray area. It is a fork that we maintain, that we don't appear to sync. However, it also overrides the name space of a non-internal package. If you think it's cut and dry that it should be included, happy to make the change. However, if you also feel that it's in a gray area, maybe it warrants more discussion and shouldn't be considered blocking for this PR.

Reply:
After talking offline I agree we should solve this in #8409